src/sage/knots/link.py

```diff
@@ -648,7 +663,20 @@ def braid(self):
             sage: A = Link([[[1, 2, -2, -1, -3, -4, 4, 3]], [1, 1, 1, 1]])
             sage: A.braid()
             s0*s1*s2*s3
+
+        Check that `PR 36884 <https://github.com/sagemath/sage/pull/36884>`__ is solved::
+
```

Comment:
```suggestion
        Check that :issue:`36884` is solved::

```

Reply:
Travis, will this cover pull requests, too?

Reply:
Yes it will.